snap/hooks/pre-refresh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash -e
 
-# save this revision for when we run again in the post-refresh
+# save these for use in the post-refresh hook
 snapctl set lastrev="$SNAP_REVISION"
-
-snapctl set release="jakarta"
+snapctl set lastver=$SNAP_VERSION
```

Comment:
Replace outdated `release` with the problematically set `lastver`. This isn't needed for this fix, but a technical dept worth getting rid of. The value of `lastver` could be used in future to implement migration logic between major releases.

Reply:
For clarity outside of the refresh hooks context, I've refactored as bellow:
```
lastrev -> pre-refresh.revision
lastver -> pre-refresh.version
```

The options will then be store as:
```
$ snap get edgexfoundry pre-refresh -d
{
	"pre-refresh": {
		"revision": 3915,
		"version": "2.3.0-dev.46"
	}
}
```

I also made sure we continue reading the old `lastrev` to handle the config file installation when refreshing from older revisions.